src/components/backend-ai-session-list.ts

```diff
@@ -1884,18 +1884,10 @@ export default class BackendAISessionList extends BackendAIPage {
         });
         document.dispatchEvent(event);
       })
-      .catch((err) => {
+      .catch(() => {
         this._selected_items = [];
         this._clearCheckboxes();
         this.terminateSessionDialog.hide();
-        this.notification.text = PainKiller.relieve(
-          'Problem occurred during termination.',
-        );
-        this.notification.show(true, err);
-        const event = new CustomEvent('backend-ai-resource-refreshed', {
-          detail: 'running',
-        });
-        document.dispatchEvent(event);
```

Comment:
The `_terminateSessionWithCheck` function uses the `_terminateKernel` function internally. The catch statement is the error catching syntax for the `_terminateKernel` function, and if an error occurs inside that function, it outputs the error to the notification. Therefore, I removed that code to avoid duplication of the notification. (No deduplication processing because no logs exist).

@yomybaby